app/src/main/java/org/feature/fox/coffee_counter/ui/ItemsView.kt

```diff
@@ -1,20 +1,164 @@
+        Item(id = "b", name = "beer", amount = 42, price = 4.99),
+        Item(id = "c", name = "mate", amount = 1337, price = 9.99))
+        ItemList(items)
+}
+
+@Composable
+fun ItemList(items: List<Item>) {
+    Column{
+        SearchBar()
+        Text(
+            stringResource(R.string.item_list_title),
+            fontWeight = FontWeight.Medium,
+            fontSize = 30.sp,
+            modifier = Modifier.padding(start = 5.dp)
+        )
+        Column(
+            modifier = Modifier
+                .verticalScroll(rememberScrollState())
+                .padding(5.dp)
+                .fillMaxWidth()
+                .weight(1f),
+            verticalArrangement = Arrangement.spacedBy(12.dp)
+        ) {
+            items.forEach { item ->
+                ItemRow(item)
+                Divider(
+                    color = Color.Gray,
+                    modifier = Modifier
+                        .fillMaxWidth(),
+                    thickness = 1.dp
+                )
+            }
+        }
+        BuyButton(amount = 55.0)
+    }
+}
+
+@Composable
+fun ItemRow(item: Item){
+    var buyItems by remember { mutableStateOf(0) }
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.SpaceBetween,
+        modifier = Modifier
+            .fillMaxWidth()
+    ) {
+        Column {
+            Text(item.name, fontWeight = FontWeight.Medium)
+            Text("${item.price}€", color = Color.Gray)
+        }
+
+        Column{
+            Row(
+                horizontalArrangement = Arrangement.spacedBy(10.dp),
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+
+                Text(
+                    "$buyItems/${item.amount}",
+                    fontWeight = FontWeight.Medium,
+                    modifier = Modifier.width(60.dp)
+                )
+
+                Button(onClick = {if (buyItems < item.amount) buyItems++},
+                    modifier = Modifier.size(35.dp),
+                    shape = CircleShape,
+                    contentPadding = PaddingValues(0.dp),
+                ) {
+                    Icon(Icons.Default.Add ,contentDescription = "content description", tint=Color.White)
+                }
+
+                Button(onClick = {if (buyItems > 0) buyItems--},
+                    modifier = Modifier.size(35.dp),
+                    shape = CircleShape,
+                    contentPadding = PaddingValues(0.dp),
+                ) {
+                    Icon(Icons.Default.Remove ,contentDescription = "content description", tint=Color.White)
+                }
+            }
+        }
     }
 }
+
+@Composable
+fun BuyButton(amount: Double){
+    Button(onClick = {},
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(bottom = 56.dp) // looks like that is the height of the NavBar. No idea how to make it better right now
```

Comment:
This right here looks like a temporary workaround. We could either fix this in this PR or create a seperate issue for this, but we need to make it work correctly.

Reply:
See #37.